docs/execution.rst

```diff
@@ -107,6 +137,31 @@ behaviour.
         };
         ExecutionStrategy executionStrategy = new AsyncExecutionStrategy(handler);
 
+Serializing results to JSON
+---------------------------
+
+The most common way to call graphql is over HTTP and to expect a JSON response back.  So you need to turn an
+`graphql.ExecutionResult` into a JSON payload.
+
+A common way to do that is use a JSON serialisation library like Jackson or GSON.  However exactly how they interpret the
+data result is particular to them.  For example `nulls` are important in graphql results and hence you must set up the json mappers
+to include them.
+
+To ensure you get a JSON result that confirms 100% to the graphql spec, you should called `toSpecification` on the result and then
```

Comment:
typo: "you should called" -> "you should call"

Reply:
fix